data/Sri Dasam Granth/0037.json

```diff
@@ -1073,7 +1073,7 @@
         "pronunciation_information": null,
         "type": "Pankti",
         "gurmukhi": {
-          "SGPC": "AqR ky clXw, iCqR CqR ky DrXw; CqR DwrIE ky ClXw, mhw sqRn ky swl hYN ]"
+          "SGPC": "AqR ky cl`Xw, iCq CqR ky Dr`Xw; CqR DwrIEN ky Cl`Xw, mhw sqRn ky swl hYN ]"
```

Comment:
```suggestion
          "SGPC": "AqR ky cl`Xw, iCq CqR ky Dr`Xw; CqR DwrIE ky Cl`Xw, mhw sqRn ky swl hYN ]"
```

I think it's without bindi, though it is tough to tell. Is there a second image for the same source? 

![image](https://user-images.githubusercontent.com/14130567/150835035-9d73b246-4841-44bc-b73f-f3e099789fef.png)

Reply:
I had thought the same, but I was looking at my physical copy of Das Granthi (July 2014).

![image](https://user-images.githubusercontent.com/4297171/150847540-2db41c6c-b0bc-4f2e-8da6-c7b842b78162.jpeg)